core-services/prow/02_config/_config.yaml

```diff
@@ -1119,13 +1112,20 @@ tide:
     - openshift/tektoncd-pipeline-operator
     - openshift/tektoncd-catalog
     - openshift/tektoncd-triggers
+    - openshift/ci-chat-bot
     - openshift/ci-ns-ttl-controller
     - openshift/ci-search
     - openshift/ci-secret-mirroring-controller
     - openshift/ci-tools
     - openshift/ci-vm-operator
+    - openshift/cincinnati
+    - openshift/cincinnati-graph-data
     - openshift/release
     - openshift/release-controller
+    - openshift/docker-distribution
```

Comment:
are these supposed to be alphabetized?  I guess not based on the previous content, but still a bit surprising.

Reply:
no